certified-connectors/Softools/apiDefinition.swagger.json

```diff
@@ -0,0 +1,879 @@
+            }
+          },
+          "401": {
+            "description": "",
+            "examples": {
+              "application/json": "{\n  \"Message\": \"Authorization has been denied for this request.\"\n}"
+            }
+          }
+        }
+      },
+      "delete": {
+        "description": "Archive a record. Call this before delete.",
+        "operationId": "Archive-record",
+        "summary": "Archive a record",
+        "parameters": [
+          {
+            "name": "AppIdentifier",
+            "in": "path",
+            "description": "Identifier for the App, can be found in the app Url /App/{AppIdentifier} or /{AppIdentifier}",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Identifier for the App",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "RecordID",
+            "in": "path",
+            "description": "Identifier of the record",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Identifier of the record",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "tenant",
+            "in": "header",
+            "description": "The tenant can be obtained from your system administrator",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "The tenant you want to connect too",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Response also includes all record data with field identifier as key",
+            "schema": {
+              "$ref": "#/definitions/Message"
+            }
+          },
+          "401": {
+            "description": "",
```

Comment:
@thduttonuk Same here and everywhere in the swagger

Reply:
Done